draft-thomson-http-oblivious.md

```diff
@@ -273,7 +286,7 @@ Given an HPKE context `context`, a request message `request`, and a response
 follows:
 
 1. Export a secret `secret` from `context`, using the string "response" as a
-   label. The length of this secret is `Nsk` - the length of the secret
+   label. The length of this secret is `Nk` - the length of an AEAD key
```

Comment:
I don't think that this is right.  Can you explain your reasoning?

Reply:
Sure: Nsk is the size of the KEM shared secret, not the AEAD encryption key. IIUC there's no difference between extracting the necessary amount and then stretching it (as we do here) or extracting something much larger and stretching.

Reply:
Oh, right.  I considered that, but didn't want to use that as encapsulation can make the value much larger than we might need.  X25519 is ok (32 bytes), but a P-256 share is larger, if only so that it can distinguish between compress and uncompressed.  Nsk might be larger in the sense that it contains wasted bits, but at least it is representative of the entropy level we are looking to maintain.

Reply:
My point is that the entropy we need is max(|key|, |nonce|), and Nk is exactly that for all AEADs I know of.

Reply:
OK, fair enough.  In TLS we never reduced the entropy of a secret below that of the KDF, which this would do.  

Anyway, could we say `max(Nk, Nn)` instead?

Reply:
Yep, done.